api/mysagw/identity/serializers.py

```diff
@@ -419,3 +419,19 @@ class MyMembershipsSerializer(MembershipSerializer):
         "role": MembershipRoleSerializer,
         "organisation": MyOrgsSerializer,
     }
+
+
+class OrganisationAdminMembersSerializer(MembershipSerializer):
+    included_serializers = {
+        "identity": PublicIdentitySerializer,
+        "role": MembershipRoleSerializer,
+        "organisation": IdentitySerializer,
```

Comment:
We don't want to give access to the `IdentitySerializer`. Why not just user the `PublicIdentitySerializer`. Maybe it needs to be expanded with `organisation_name`.

Reply:
Done